Microsoft.Toolkit.Uwp.SampleApp/SamplePages/OneDrive Service/OneDriveCode.bind

```diff
@@ -0,0 +1,96 @@
+
+// OneDriveService works for OneDrive Consumer as well as OneDrive For Business (Office 365)
+//
+// Authentication 
+// In order to use the OneDriveService you need to authenticate the user and get an access token
+
+// OneDrive consumer you have two options:
+// 1) If the user is connected to a windows session with a Microsoft Account, the service is able to silently get an access token.
```

Comment:
1) Several examples of this, but "windows" should be "Windows".
2) Ensure consistency in using "Microsoft account" and "MS Account".